assets/js/load-chart.js

```diff
@@ -166,7 +169,8 @@ function setupChart () {
                 'beforeend',
                 `<tr class="${idx % 2 ? 'odd' : 'even'}">
                     <td>${scores.focus_areas[area]}</td>
-                    <td class="score">${String(recent_score[idx + AREA_SCORE_OFFSET] / 10).padEnd(4, '.0')}%</td>
+                    <td class="score">${String(recent_score[idx + AREA_SCORE_OFFSET].per_mille / 10).padEnd(4, '.0')}</td>
```

Comment:
The "score" column seems to have lost the `%` symbol here (see also: the screenshot in the PR description). I think we should keep it.

Reply:
This is recovered in the newer version.